canonical/ubuntu-advantage-client
# cis: post enable message pointing to readme

## Discussion

The exact content here still needs discussion and approval before merging.

## Proposed Commit Message


> cis: post enable message pointing to readme
> 
> After enabling cis using ua, the user needs to
> take additional steps to use the cis packages
> that were installed. This adds messaging to point
> users in the right direction to get started.

## Test Steps

Enable CIS. Follow the messages to look at the readmes and verify that the messages pointed you to the correct locations.

## Desired commit type::


## Checklist:
